engine/apps/api/views/alert_receive_channel.py

```diff
@@ -474,6 +474,7 @@ def filters(self, request):
         api_root = "/api/internal/v1/"
 
         filter_options = [
+            {"name": "search", "type": "search"},
```

Comment:
adding the search filter explicitly here (and in multiple other places) because of this frontend change: https://github.com/grafana/oncall/pull/4713/files#diff-30e131a5ff3d32492f7cc4f5701a929836d5a016286cbf0d28cad96ec2e18978L61-L65